doc/command-t.txt

```diff
@@ -774,6 +774,11 @@ Following is a list of all available options:
         work on systems without the tool or with an incompatible version of
         the tool.
 
+      - "rg": uses the command-line tool of the same name, also known as
```

Comment:
https://github.com/BurntSushi/ripgrep would be nice to paste this link into the documentation

Reply:
i guess this is probably not necessary because the full name ripgrep is there, so googling  is posssible